registry.json

```diff
@@ -3382,5 +3382,20 @@
     "telegram": "yadavnie",
     "operatorName": "Yadav/Nadeem",
     "github": "adamoonpool"
+  },
+  {
+    "ticker": "ISTAN",
+    "poolId": "bce4f15e81e2d2d9fe5c061f02f89ced9a56654a753f14d8409505bb",
+    "social": {
+      "facebook": "",
+      "twitter": "istanpool34",
+      "website": "https://istanpool.info",
+      "reddit": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "istanpool.info",
```

Comment:
Hey I cant find this telegram handle. Please can you make sure this is a telegram handle of an individual user and not a group or channel? Thanks

Reply:
Hi there, 
Thanks for reaching to me. 
do you mean this: https://t.me/istanpool or my telegram username is @istanpool

Reply:
`istanpool` is fine

Reply:
thanks, do I need to do anything else?

Reply:
Just change  the telegram handle to your telegram handle

Reply:
Sorry, I am not good at using GitHub, I hope did it correctly.

Reply:
the file was updated.